kordac/processors/errors/TagNotMatchedError.py

```diff
@@ -0,0 +1,15 @@
+from kordac.processors.errors import Error
+
+class TagNotMatchedError(Error):
```

Comment:
This would be if a start tag does not find an end tag or an end tag is found without a start tag. Message attribute should be used to clarify which specific case.